pkg/splunk/client/minioclient.go

```diff
@@ -47,8 +47,7 @@ func NewMinioClient(bucketName string, accessKeyID string, secretAccessKey strin
 	var s3SplunkClient SplunkMinioClient
 	var err error
 
-	region := GetRegion(endpoint)
-	cl := fn(region, accessKeyID, secretAccessKey)
+	cl := fn(endpoint, accessKeyID, secretAccessKey)
```

Comment:
@jryb this is something which I noticed. Was it working before? And I hope this change is right now?

Reply:
It was working before.  Maybe get region was failing and just returning `endpoint`.  I'll verify minio once this is merged as part of the minio ut work.